src/OSInfo.cpp

```diff
@@ -234,46 +258,103 @@ QString OSInfo::getOSVersion() {
 	           static_cast<unsigned long>(minor),
 	           static_cast<unsigned long>(bugfix),
 	           buildno ? buildno : "unknown");
+
+	return os;
+}
+#else
+QString OSInfo::getOSVersionOther() {
+#ifdef Q_OS_LINUX
+	QString os = getOSVersionLinux();
+	if (os.isEmpty()) {
```

Comment:
Is there a specific reason to use `isEmpty` here wherease everywhere else it's using `isNull`?

Reply:
Yes, `isNull()` returns `true` when the string has never been set.

Reply:
In that case `isEmpty` should also return `true`, shouldn't it? From what I have understood, the real difference between the two is that `isNull` returns `false` for something like `QString("")`.

I really only commented on this for consistency reasons. In all other occasions you have used `isNull` to check whether the os-string has been set by the method. Only here you use `isEmpty`. And the way I understood the code, `isNull` should work in this case as well. It's not really that much of a deal though...